packages/errors/src/messages.ts

```diff
@@ -35,4 +44,23 @@ export const SolanaErrorMessages: Readonly<{
     [SOLANA_ERROR__TRANSACTION_SIGNATURE_NOT_COMPUTABLE]:
         "Could not determine this transaction's signature. Make sure that the transaction has " +
         'been signed by its fee payer.',
+    [SOLANA_ERROR__TRANSACTION_INVOKED_PROGRAMS_CANNOT_PAY_FEES]:
+        'This transaction includes an address (`$programAddress`) which is both ' +
+        'invoked and set as the fee payer. Program addresses may not pay fees',
```

Comment:
I'm not 100% sure if this error should be this, or:


```suggestion
    [SOLANA_ERROR__TRANSACTION_PROGRAMS_CANNOT_PAY_FEES]:
        'This transaction includes a program (`$programAddress`) which is set as ' +
        'the fee payer. Program addresses may not pay fees',
```

ie. does it matter that it's invoked, or is it that programs can't pay fees, period? cc/ @joncinque